addons/web_editor/static/src/js/editor/odoo-editor/src/utils/utils.js

```diff
@@ -1585,7 +1585,11 @@ export function isUnbreakable(node) {
                 node.getAttribute('t-value') ||
                 node.getAttribute('t-out') ||
                 node.getAttribute('t-raw'))) ||
-        node.classList.contains('oe_unbreakable')
+        node.classList.contains('oe_unbreakable') ||
+        (node.tagName === "A" &&
+            (node.classList.contains("btn") ||
+                node.getAttribute("role") === "tab" ||
+                node.getAttribute("role") === "button"))
```

Comment:
```suggestion
        node.matches(".oe_unbreakable, a.btn, a[role='tab'], a[role='button']");
```

Reply:
Thanks @bvr-odoo for suggestion, I did changes on behalf of @ksbh-odoo